tests/x11/vnc_two_passwords.pm

```diff
@@ -101,7 +101,7 @@ sub generate_vnc_events {
 
     send_key 'alt-f4';
     wait_serial('vncviewer-finished') || die 'vncviewer not finished';
-    type_string "exit \n";
+    enter_cmd "exit ";
 }
```

Comment:
It could be `enter_cmd "exit";` (removed extra space).

Reply:
done